### PR TITLE
Update MC version manifest url

### PR DESCRIPTION
**Pull request by kennytv**

### CODE REVIEW

**Comment by NoahvdAa** [comment]
The create paperclip jar task should probably also be updated to use `piston-data.mojang.com`

**Review by NoahvdAa — APPROVED** [review]

File: paperweight-lib/src/main/kotlin/tasks/CreatePaperclipJar.kt [hunk 1]
```diff
@@ -104,7 +104,7 @@ abstract class CreatePaperclipJar : JavaLauncherZippedTask() {
         val originalJar = originalBundlerJar.path
         val vanillaSha256Hash = originalJar.sha256asHex()
         val vanillaSha1Hash = toHex(originalJar.hashFile(digestSha1))
-        val vanillaUrl = "https://launcher.mojang.com/v1/objects/$vanillaSha1Hash/server.jar"
+        val vanillaUrl = "https://piston-data.mojang.com/v1/objects/$vanillaSha1Hash/server.jar"
```
Comment (jpenilla) [inline paperweight-lib/src/main/kotlin/tasks/CreatePaperclipJar.kt:107]: this should maybe use the URL from the version manifest similar to the bundler download task in InitialTasks

**Review by jpenilla — CHANGES_REQUESTED** [review]

**Comment by jpenilla** [comment]
Needs testing to check old dev bundles still work on new userdev and vice-versa, should be fine but we need to be sure.

File: paperweight-lib/src/main/kotlin/tasks/CreatePaperclipJar.kt (hunk 1 above)
Comment (jpenilla) [reply paperweight-lib/src/main/kotlin/tasks/CreatePaperclipJar.kt:107]: This can possibly be addressed at a later date

**Review by jpenilla — APPROVED** [review]

**Pull request merged**